docs/Developers/Newcomers_Guide.rst

```diff
@@ -383,6 +383,25 @@ and clicking ``New Pull Request``.
 
 **Congratulations!** You have now created your first ``Pull Request``!
 
+Now, before having other contributors review your work, the system will do an
+automated code review and will comment on your Pull Request.
```

Comment:
This should `will comment the mistakes on your Pull Request` imo. We can also add a line of what kind of mistakes the system will show. like commit messages, styling issues etc.

Reply:
I could do something like: `will comment the mistakes(coding style errors, neglecting commit message rules, line length and spacing etc.)on your Pull Request.`